flatcar-terraform-hetzner/README.md

```diff
@@ -0,0 +1,67 @@
+# Flatcar Provisioning Automation for Hetzner Cloud
+
+This repository provides tools to automate Flatcar provisioning on [Hetzner Cloud](https://www.hetzner.com/cloud) using [Terraform](https://www.terraform.io/).
+
+## Features
+
+- Minimal configuration required (demo deployment works with default settings w/o any customisation, just run `terraform apply`!).
+- Deploy one or multiple servers.
+- Per-server custom configuration via separate [container linux config](https://www.flatcar.org/docs/latest/provisioning/config-transpiler/configuration/) files.
+
+
+## Prerequisites
+
+1. A Hetzner Cloud [API token](https://docs.hetzner.com/cloud/api/getting-started/generating-api-token/).
+
+
+## HowTo
+
+This will create a server in Falkenstein using the smallest instance size available.
+See "Customisation" below for advanced settings.
+
+1. Clone the repo.
+2. Run
+   ```shell
+   terraform init
```

Comment:
I think the HCLOUD_TOKEN export should go first (with saying that it's optional and there will be a prompt).

Reply:
Done.

Reply:
(before `terraform plan`, it's not needed for `init`)